ui/src/components/containers/taskBox.tsx

```
@@ -0,0 +1,109 @@
+            {props.taskDetails.image}
+          </p>
+          <p>
+            <strong class="font-medium text-gray-300">Parameters:</strong>
+          </p>
+          <ul class="ml-4 list-disc text-gray-400">
+            {props.taskDetails.parameters &&
+              props.taskDetails.parameters.map((param) => (
+                <li>
+                  {param}
+                </li>
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)